### ISSUE TITLE
Add A function to automatically rebuild all Shader Caches and PPTC Caches automatically 

### ISSUE THREAD

**Issue by Shahil-Ayato** [Ryujinx/Ryujinx#2102]
#### Overview: 

- Add a button to automatically rebuild all shaders and PPTC caches for every game. 

#### Smaller Details: 
- should be put in the General settings. 

#### Nature of Request:  
- Addition/Change
  - this is similar to RPCS3’s PPU cache button where it rebuilds every games PPU cache 
 

### Why would this feature be useful?
This would be incredibly useful for after a driver update or  if you use LDN builds and switch back to master a lot or if you test a lot of WIP Pull request builds. Because you need to rebuild your shader cache every driver update and it can take a long time for just a single game, to add onto that if you go from a WIP Pull Request build and then back to master (same thing with LDN) your required to rebuild PPTC and your shader Cache. This can take a significantly long time especially if you have to do it for every game separately and click on every single one to boot them.

**Comment by Powerlated** [comment]
I'll take this as my first contribution, could a dev assign me? Thanks.

**Comment by gdkchan** [comment]
Done.

**Comment by DCaedus** [comment]
Yes please, or an option to fully build Shader cache at the start of each game like Dolphin, this would solve the shader cache stuttering for good.
